crypto/pkcs12/pk12err.c

```diff
@@ -17,7 +17,7 @@
 static const ERR_STRING_DATA PKCS12_str_reasons[] = {
     {ERR_PACK(ERR_LIB_PKCS12, 0, PKCS12_R_CALLBACK_FAILED), "callback failed"},
     {ERR_PACK(ERR_LIB_PKCS12, 0, PKCS12_R_CANT_PACK_STRUCTURE),
-    "cant pack structure"},
+    "can't pack structure"},
```

Comment:
This needs to be changed also here: https://github.com/openssl/openssl/blob/master/crypto/err/openssl.txt#LL925C1-L925C29

Reply:
These I had changed in the first place, but had been told **not** to change them because they are autogenerated
(https://github.com/openssl/openssl/pull/20910#discussion_r1188433626 and https://github.com/openssl/openssl/pull/20910#discussion_r1188433785).

Reply:
This is autogenerated so should revert this I think